From 1ca391480e86bf7eaac1ecb0f74cb9e926f9675e Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Thu, 21 Dec 2017 08:21:02 +0900
Subject: [PATCH 644/909] arm64: dts: r8a7795: Add QoS device node

commit d75b99734e86e491ea0ce48f7526de154952a020 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds QoS device node for the R8A7795 SoC.

[takeshi.kihara.df: fix for warning when compiling with W=1:
 Node /soc/qos@e67e0000 has a unit name, but no reg property]
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795.dtsi b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
index ae04d3461fc0..4d794ef4113b 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
@@ -1598,6 +1598,11 @@
 			#iommu-cells = <1>;
 		};
 
+		qos@e67e0000 {
+			compatible = "renesas,qos";
+			reg = <0 0xe67e0000 0 0x10090>;
+		};
+
 		avb: ethernet@e6800000 {
 			compatible = "renesas,etheravb-r8a7795",
 				     "renesas,etheravb-rcar-gen3";
-- 
2.17.1

